### DISCUSSION TITLE
open a single file automatically

### DISCUSSION THREAD

**Discussion by kqvanity** [olimorris/persisted.nvim#159] [Q&A]
I want to open single files with the previous session state they were at. The docs mention
> By design, the plugin will not autoload a session when any arguments are passed to Neovim such as nvim my_file.py

Conventionally, I open sessions by landing on nvim dashboard during startup `nvim` then choose one from telescope, but at other times, I'd open a single file directly (its session doesn't load until I isssue the command `SessionLoadLast`

**Comment by olimorris** [comment]
Have you looked at the examples in the readme? There's a lot you can do to extend the functionality of the plugin.

**Comment by kqvanity** [comment]
I did read it (hence the readme excerpt) The snippet provided allows loading the session encompassing the file, but I want to reload the session of separate individual files. am I missing on something?
